### ISSUE TITLE
Problems passing add-opens flags

### ISSUE THREAD

**Issue by headius** [jruby/jruby-launcher#27]
In https://github.com/jruby/jruby/issues/5736#issuecomment-530313123 @janko reported that on Travis, we are still failing to pass the proper `add-opens` flags that would allow `sun.nio.ch` to be opened up for `FilenoUtil` use.

It appears that the flags are there, but for some reason not being applied in some environments.

**Comment by headius** [comment]
Ok it looks like we made the appropriate changes on jruby-launcher master (in part thanks to @jeremyevans in #24...and then never pushed an updated gem release!

I've pushed jruby-launcher 1.1.10 now.

@janko Can you try this in Travis?

@keithrbennett This is probably the actual reason why you saw that warning. Can you try to reinstall JRuby with rvm and see if it works properly now?

**Comment by janko** [comment]
It worked, we're [green](https://travis-ci.org/shrinerb/shrine/jobs/583588688) now 🎉

**Comment by headius** [comment]
Note that there are additional improvements to the bash script coming in jruby/jruby#5826 as well as the improvements to accessibility checking I already merged in jruby/jruby#5843.

The bash script changes will be incorporated into the native launcher as soon as possible.

**Comment by keithrbennett** [comment]
@headius I just installed jruby-head, and ran it again, but got the same warnings. Did I do something wrong?:

```
➜  gitlab-sandbox git:(master) ✗   rvm current; jruby -e nil
jruby-head
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.headius.backport9.modules.Modules to method sun.nio.ch.NativeThread.signal(long)
WARNING: Please consider reporting this to the maintainers of com.headius.backport9.modules.Modules
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
➜  gitlab-sandbox git:(master) ✗   ruby -v
jruby 9.2.9.0-SNAPSHOT (2.5.3) 2019-09-13 4dc9eca OpenJDK 64-Bit Server VM 11.0.1+13 on 11.0.1+13 +jit [darwin-x86_64]```
